lib/carin_for_blue_button_test_kit/client/v2.0.0/mock_server.rb

```diff
@@ -113,12 +127,18 @@ def remove_transfer_encoding_and_content_length_header(headers)
 
     def match_request_to_expectation(resource_type, params)
       matched_search = SEARCHES_BY_PRIORITY[resource_type.to_sym].select do |expectation|
-        (params.keys.map(&:to_s) & expectation).sort == expectation
+        (params.keys.map(&:to_s).map do |key|
+          if key.include?('_lastUpdated')
+            key.split('%').first
+          else
+            key
+          end
+        end & expectation).sort == expectation
       end.map(&:first)
 
       if matched_search.present?
         return params.select do |key, value|
-          matched_search.include?(key.to_s) || key == '_revInclude' || key == '_include'
+          matched_search.include?(key.to_s.split('%').first) || key == '_revInclude' || key == '_include'
```

Comment:
I think you don't need any of this fancy stuff because the modification is all in the value on the right side of the equals sign!